README.md

```diff
@@ -31,6 +31,14 @@ A frontend Framework for building admin applications running in the browser on t
 * Can be included in another React app
 * Inspired by the popular [ng-admin](https://github.com/marmelab/ng-admin) library (also by marmelab)
 
+## Versions In This Repository
+
+* [master](https://github.com/marmelab/admin-on-rest/commits/master) - commits that will be included in the next _patch_ release
+
+* [next](https://github.com/marmelab/admin-on-rest/commits/next) - commits that will be included in the next _major_ or _minor_ release
+
+Most PRs should be made to **next**.
```

Comment:
s/Most PRs should be made to **next**/Bugfix PRs that don't break BC should be made against **master**. All other PRs (new features, bugfix with BC break) should be made against **next**/